keras_hub/src/models/yolo_v8/yolo_v8_backbone_presets.py

```diff
@@ -0,0 +1,104 @@
+backbone_presets_no_weights = {
+    "yolo_v8_xs_backbone": {
+        "metadata": {
+            "description": "An extra small YOLOV8 backbone",
+            "params": 1277680,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_xs_backbone/3",
+    },
+    "yolo_v8_s_backbone": {
+        "metadata": {
+            "description": "A small YOLOV8 backbone",
+            "params": 5089760,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_s_backbone/3",
+    },
+    "yolo_v8_m_backbone": {
+        "metadata": {
+            "description": "A medium YOLOV8 backbone",
+            "params": 11872464,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_m_backbone/3",
+    },
+    "yolo_v8_l_backbone": {
+        "metadata": {
+            "description": "A large YOLOV8 backbone",
+            "params": 19831744,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_l_backbone/3",
+    },
+    "yolo_v8_xl_backbone": {
+        "metadata": {
+            "description": "An extra large YOLOV8 backbone",
+            "params": 30972080,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_xl_backbone/3",
+    },
+}
+
+backbone_presets_with_weights = {
+    "yolo_v8_xs_backbone_coco": {
+        "metadata": {
+            "description": (
+                "An extra small YOLOV8 backbone pretrained on COCO"
+            ),
+            "params": 1277680,
+            "official_name": "YOLOV8",
+            "path": "yolo_v8",
+        },
+        "kaggle_handle": "kaggle://kerashub/yolov8/keras/yolo_v8_xs_backbone_coco/3",  # noqa: E501
```

Comment:
remove these noqa marks

Reply:
The noqa marks for this file have been removed.